aem-cif-product-page-extension/extension/src/components/AfterProductData/afterProductData.js

```diff
@@ -12,8 +12,33 @@
  *
  ******************************************************************************/
 import React from 'react';
-const AfterProductData = () => {
-    return <div>This will be displayed after the product data</div>;
+import { shape, string } from 'prop-types';
+import useAemProductData from '../../talons/useAemProductData';
+import ProductImages from './productImages';
+import classes from './afterProductData.css';
+
+const AfterProductData = ({ productDetails }) => {
+    const { sku } = productDetails;
+    const [{ loading, error, productData }] = useAemProductData({ sku });
```

Comment:
`error` is declared but not used anywhere. This is done in some other places as well.